lib/extend.js

```diff
@@ -6,6 +6,21 @@ module.exports = function (Potential, $Promise) {
 
   //----- promise methods -----
 
+  $Promise.debounce = function (sourceFn) {
```

Comment:
`$Promise` is the internal promise class and is not exposed to the user — this can be placed on `Potential`, which is the library / public API namespace.